.vscode/launch.json

```diff
@@ -24,6 +24,14 @@
         "uriFormat": "%s",
         "action": "debugWithChrome"
       }
-    }
+    },
+    {
```

Comment:
So, if I understand this correctly, the `migration-tools` directory was created to find and change the files that had the `<a name` pattern, so once the script is run, we don't need it anymore, correct? So this can also be removed?

Reply:
Yes, this was a tool for that specific purpose. I thought it might be useful if we had similar problems to correct another time ,but for sake of clarity I can remove it.

Reply:
It looks like this is still in the PR.

Reply:
Now done.